### PR TITLE
new Redis module

### PR DESCRIPTION
**Pull request by briskt**

### CODE REVIEW

File: aws/elasticache/redis/main.tf [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+resource "aws_elasticache_subnet_group" "redis_subnet_group" {
+  name       = var.subnet_group_name
+  subnet_ids = var.subnet_ids
+}
+
+/*
+ * Create Cluster
+ */
+resource "aws_elasticache_cluster" "redis" {
+  cluster_id           = replace(var.cluster_id, "/(.{0,20})(.*)/", "$1")
```
Comment (Baggerone) [inline aws/elasticache/redis/main.tf]: Maybe add a comment to describe what's happening here.

**Review by Baggerone — APPROVED** [review]

File: aws/elasticache/redis/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+This module is used to create an Elasticache cluster of Redis servers
+
+## What this does
+
+ - Create Elasticache subnet group spanning all provided `var.subnet_ids`.
+ - Create Elasticache cluster of Redis servers.
+
+## Required Inputs
+
+ - `cluster_id` - ID/Name for Elasticache Cluster
```
Comment (Baggerone) [inline aws/elasticache/redis/README.md]: Maybe add a note about how it will be modified (by that regex).

**Review by fillup — APPROVED** [review]

File: aws/elasticache/redis/main.tf (hunk 1 above)
Comment (briskt) [reply aws/elasticache/redis/main.tf]: Heh, good question. I just copied it from the `memcache` module. I'll see if I can figure it out and write a comment.

Comment (briskt) [reply aws/elasticache/redis/main.tf]: I guess it's pretty simple...just takes the first 20 characters.

**Pull request merged**